server/forge/bitbucket/internal/types.go

```diff
@@ -108,15 +110,15 @@ type Repo struct {
 	Scm        string  `json:"scm"`
 	Desc       string  `json:"desc"`
 	Links      Links   `json:"links"`
-	Mainbranch struct {
+	MainBranch struct {
 		Type string `json:"type"`
 		Name string `json:"name"`
-	} `json:"mainbranch"`
+	} `json:"mainbranch"` // cspell:disable-line
```

Comment:
Can we just allow the wrong word (`mainbranch`) instead of disabling the complete line?

Reply:
well I would not do so ... as it's just a bitbucket quirks

Reply:
https://github.com/streetsidesoftware/cspell/issues/2799